customChatWidget/src/containers/ChatWidget/index.js

```diff
@@ -181,7 +181,6 @@ const ChatWidget = ({
                 "ATTACHMENTS": !!enableAttachments,  // this is the override flag from user for attachments
                 "MESSAGING_MARKDOWN": true // enable rich messaging toolbar and text formatting
```

Comment:
Does this line also need to be removed? I remember we discussed before and decided to only enable this feature from one place. I'm forgetting why both this line and below line were added

Reply:
Correct, I missed this in initial revision. Publishing new commit now